grails-forge-core/src/main/java/org/grails/forge/feature/assetPipeline/AssetPipeline.java

```diff
@@ -135,6 +135,11 @@ public String getCategory() {
 
     @Override
     public String getDocumentation() {
+        return "https://github.com/bertramdev/asset-pipeline#readme";
+    }
+
+    @Override
+    public String getThirdPartyDocumentation() {
```

Comment:
https://www.asset-pipeline.com/manual is not third-party documentation. It is first-party documentation. Keep it in `getDocumentation()` but commented out with a comment that it is not available at the moment but should be updated when back online. And return the readme for now like you have done.